### DISCUSSION TITLE
Poor Performance when the `textContainer.exclusionPaths` is set

### DISCUSSION THREAD

**Discussion by aibobrov** [krzyzanowskim/STTextView#67] [Questions]
Text layout is taking too much time. It makes huge freeze for the first frame and while scrolling both on macOS and iOS. 

Code needed to reproduce the issue
```swift
// iOS
textView.textContainer.exclusionPaths = [UIBezierPath(rect: CGRect(origin: .zero, size: CGSize(width: 100, height: 100)))]

// macOS
textView.textContainer.exclusionPaths = [NSBezierPath(rect: NSRect(origin: .zero, size: NSSize(width: 100, height: 100)))]
```

**Comment by krzyzanowskim** [comment]
I don't think I can do anything about that one in particular. It happens implicitly between `NSTextContainer` and `NSTextLayoutManager`
